patronload/config.py

```diff
@@ -1,8 +1,22 @@
 import logging
 import os
+from typing import Optional
 
 import sentry_sdk
 
+EXPECTED_CONFIG_VARIABLES = [
```

Comment:
What I would probably do instead here is set default config values and then in the `load_config_values()` function update them from ENV. If a value isn't present in ENV then this will just use the default. We don't actually need all these in every environment, so setting a default value of `None` for those is fine.